timezones/fields.py

```diff
@@ -35,6 +33,11 @@ def validate(self, value, model_instance):
     def run_validators(self, value):
         # coerce value back to a string to validate correctly
         return super(TimeZoneField, self).run_validators(smart_str(value))
+
+    def from_db_value(self, value, expression, connection, context):
+        if value is None:
+            return value
```

Comment:
would this be clearer if it did `return None`?

Reply:
I'm actually considering that the `coerce_timezone_value` function in `utils` should be able to deal with a `value==None`.

Reply:
`return coerce_timezone_value(value) if value is not None else None` ?